### PR TITLE
Implement updated spacing for the column picker

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-column-picker-item-spacing](https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-column-picker-item-spacing)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-column-picker-item-spacing](https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-column-picker-item-spacing)

**Review by Blackbaud-ToddRoberts — DISMISSED** [review]

File: src/app/visual/data-manager/data-manager-visual.component.ts [hunk 1]
```diff
@@ -116,7 +121,8 @@ export class DataManagerVisualComponent implements OnInit {
   public settingsKey = 'test';
 
   constructor(
-    private dataManagerService: SkyDataManagerService
+    private dataManagerService: SkyDataManagerService,
+    public themeSvc: SkyThemeService
```
Comment (Blackbaud-AlexKingman) [inline src/app/visual/data-manager/data-manager-visual.component.ts:125]: Shouldn't this be `@Optional() private themeSvc?: SkyThemeService`?

File: src/app/visual/data-manager/data-manager-visual.component.html [hunk 2]
```diff
@@ -1,3 +1,8 @@
+<sky-docs-demo-control-panel-theme
+(themeSettingsChange)="themeSettingsChange($event)"
```
Comment (Blackbaud-AlexKingman) [inline src/app/visual/data-manager/data-manager-visual.component.html:2]: Missing indentation here.

**Review by Blackbaud-AlexKingman — COMMENTED** [review]
Two minor things.

File: src/app/visual/data-manager/data-manager-visual.component.html (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/visual/data-manager/data-manager-visual.component.html]: Fixed

File: src/app/visual/data-manager/data-manager-visual.component.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/visual/data-manager/data-manager-visual.component.ts]: Done

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-column-picker-item-spacing](https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-column-picker-item-spacing)

**Pull request merged**